README.md

```diff
@@ -128,9 +128,7 @@ To run style checking run `make style`.
 
     * get_repos.py -> Downloads the repos list.
 
-    * delete_cache_entries.py -> Delete cache entries with certain results.
-
-    * delete_inconsistent_merge_results.py -> Delete cache entires with inconsistent merge results.
+    * compare_dirs.py -> Compares two directories.
```

Comment:
I see compare_dirs.py defined (though I can't locate it in the GitHub diffs in orderto leave a comment on it...).  However, I do not see it used anywhere.  I think it can be replaced by `diff -q -r dir1 dir2 2>&1 /dev/null`.